group.yml

```diff
@@ -23,9 +23,6 @@ multi_arch:
 
 operator_image_ref_mode: manifest-list
 
-# wip see: issues.redhat.com/browse/ART-3107
-operator_channel_stable: default
```

Comment:
If you remove this definition but leave the references to `operator_channel_stable` [here](https://github.com/openshift-eng/doozer/blob/0a73132bc3e97901f0651d99463c47e4e8267b1e/doozerlib/olm/bundle.py#L576-L579), I think you'll end up with a python error like `object has no attribute 'operator_channel_stable'`?

If that's true, then you could work around it by changing the value of this variable instead.

Reply:
if so, this part of logic https://github.com/openshift-eng/doozer/blob/master/doozerlib/olm/bundle.py#L576-L579 can be removed